README.md

```diff
@@ -1,16 +1,42 @@
-# flexmeasures-openweathermap - a plugin for FlexMeasures
-
+# flexmeasures-openweathermap - a plugin for FlexMeasures to integrate OpenWeatherMap data
 
 ## Usage
 
+To register a new weather sensor:
+
+`flexmeasures owm register-weather-sensor --name wind-speed --latitude 30 --longitude 40`
+
+Currently supported: wind_speed, temperature & radiation.
+
+Notes about weather sensor setup: 
+
+- Weather sensors are public. They are accessible by all accounts on a FlexMeasures server. TODO: maybe limit this to a list of account roles.
+- The resolution is one hour. OWM also supports minutely data within the upcoming hour(s), but that is not supported here.
+
+To collect weather forecasts:
+
+`flexmeasures owm get-weather-forecasts --location 30,40`
+
+This saves forecasts for your registered sensors in the database.
+
+Use the `--help`` option for more options, e.g. for specifying two locations and ask for a few weather stations to cover the region between them.
+
+An alternative usage is to save raw results in JSON files (for later processing), like this:
+
+`flexmeasures owm get-weather-forecasts --location 30,40 --store-as-json-files --region somewhere`
+
+This saves the complete response from OpenWeatherMap in a local folder (i.e. no sensor registration needed, this is a direct way to use OWM, without FlexMeasures integration). `region` will become a subfolder.
```

Comment:
Yes. It's like a tag. We thought we'd also tag weather data in the db one day, but that seems not what will happen.

In the db, it could become the name of the weather station. Actually, this might be nicer than passing in an asset ID (instead of the exact location, which is a TODO item).

Q: should we make a `UniqueConstraint` on `GenericAsset.name` and `GenericAsset.account_id`? I already added such a constraint in the Schema for `GenericAsset`.

Reply:
A: That sounds reasonable.

Reply:
Oh wait, we already have that constraint:

```

    __table_args__ = (
        UniqueConstraint(
            "name",
            "account_id",
            name="generic_asset_name_account_id_key",
        ),
    )
```